charts/k8ssandra/README.md

```diff
@@ -42,14 +42,12 @@ Provisions and configures an instance of the entire K8ssandra stack. This includ
 | cassandra.cassandraLibDirVolume.storageClass | string | `"standard"` | Storage class for persistent volume claims (PVCs) used by the underlying cassandra pods. Depending on your Kubernetes distribution this may be named "standard", "hostpath", or "localpath". Run `kubectl get storageclass` to identify what is available in your environment. |
 | cassandra.cassandraLibDirVolume.size | string | `"5Gi"` | Size of the provisioned persistent volume per node. It is recommended to keep the total amount of data per node to approximately 1 TB. With room for compactions this value should max out at ~2 TB. This recommendation is highly dependent on data model and compaction strategies in use. Consider testing with your data model to find an optimal value for your usecase.  |
 | cassandra.allowMultipleNodesPerWorker | bool | `false` | Permits running multiple Cassandra pods per Kubernetes worker. If enabled resources.limits and resources.requests **must** be defined. |
-| cassandra.heap | object | `{"newGenSize":null,"size":null}` | Cluster-level heap configuration |
-| cassandra.resources | object | `{}` | Resource requests for each Cassandra pod. |
+| cassandra.resources | object | `{}` | Cluster-level heap configuration (optional). See reference: https://k8ssandra.io/docs/reference/k8ssandra/#values for additional information. size: newGenSize: -- Resource requests for each Cassandra pod. |
```

Comment:
This is out place. `resources` is for setting the pod's cpu and/or memory.

Reply:
Right, this goes to @jakerobb's comment about the comment getting merged into the next field. Removing the `--` fixes the issue, but the comment is not rendered in the docs (which may be what we want).